livekit-api/src/services/sip.rs

```diff
@@ -58,18 +60,24 @@ pub struct CreateSIPTrunkOptions {
 #[derive(Default, Clone, Debug)]
 pub struct CreateSIPInboundTrunkOptions {
     /// Optional free-form metadata.
-    pub metadata: String,
+    pub metadata: Option<String>,
```

Comment:
these aren't required, updating them to optional

Reply:
Generally using default values is fine in this case, but an Option is ok too

Reply:
yeah, it's easier for users if they didn't have to fill out every field as default